### PR TITLE
Fix: Share button appears unshared first

### PR DESCRIPTION
**Pull request by cballevre**
Refactor code to group share button and recipients. This correction make also recipients clickable.

**Related PRs:**
- https://github.com/cozy/cozy-libs/pull/1874
- https://github.com/cozy/cozy-ui/pull/2286

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -70,8 +70,8 @@
     "cozy-logger": "^1.9.0",
     "cozy-realtime": "4.0.5",
     "cozy-scripts": "6.3.8",
-    "cozy-sharing": "^4.5.10",
-    "cozy-ui": "^70.6.0",
+    "cozy-sharing": "^4.7.0",
+    "cozy-ui": "^77.6.0",
```
Comment (Crash--) [inline package.json:74]: no BC to deal with?

Comment (cballevre) [reply package.json:74]: sry, but what do you mean by BC ?

Comment (Crash--) [reply package.json:74]: breaking changes. https://github.com/cozy/cozy-ui/blob/master/CHANGELOG.md between 70 and 76. Notes is not concerned by them?

Comment (cballevre) [reply package.json:74]: Notes is not concerned because it doesn't use `List`, `Dropdown`, `BottomSheet`from my research.

**Review by Crash-- — APPROVED** [review]

**Pull request merged**